tests/unit/lms/services/oauth2_token_test.py

```diff
@@ -62,9 +62,7 @@ def oauth_token_in_db_or_not(
         oauth_token = factories.OAuth2Token.build(
             user_id=lti_user.user_id,
             consumer_key=application_instance.consumer_key,
-            # Don't link to an application instance directly, or factoryboy
```

Comment:
Not 100% sure but I'd say this was caused now a problem due to the first bullet point on:

https://github.com/sqlalchemy/sqlalchemy/releases/tag/rel_1_4_21

In any case it only affected one test and session handling in the tests with fixtures and factories can sometimes lead to some very rare  edge cases.